v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/source/reader/io/jdbc/rowmapper/JdbcSourceRowMapper.java

```diff
@@ -103,7 +102,8 @@ long getCurrentTimeMicros() {
                     entry.getKey(),
                     this.mappingsProvider
                         .getMappings()
-                        .getOrDefault(entry.getValue().getName(), JdbcValueMapper.UNSUPPORTED)
```

Comment:
[nit] extra new line in the diff.

Reply:
This was auto-updated by `spotless`, since I added `.toUpperCase` and it no longer fits in a single line